packages/block-library/src/accordion-group/index.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * Server-side rendering of the `core/accordion-group` block.
+ *
+ * @package WordPress
+ * @since 6.6.0
+ *
+ * @param array $attributes The block attributes.
+ * @param string $content The block content.
+ *
+ * @return string Returns the updated markup.
+ */
+
+function render_block_core_accordion_group( $attributes, $content ) {
+	if ( ! $content ) {
+		return $content;
+	}
+
+	$suffix = wp_scripts_get_suffix();
+	if ( defined( 'IS_GUTENBERG_PLUGIN' ) && IS_GUTENBERG_PLUGIN ) {
+		$module_url = gutenberg_url( '/build/interactivity/accordionGroup.min.js' );
+	}
+
+	wp_register_script_module(
+		'@wordpress/block-library/accordion-group',
+		isset( $module_url ) ? $module_url : includes_url( "blocks/accordion-group/view{$suffix}.js" ),
+		array( '@wordpress/interactivity' ),
+		defined( 'GUTENBERG_VERSION' ) ? GUTENBERG_VERSION : get_bloginfo( 'version' )
+	);
+
+	wp_enqueue_script_module( '@wordpress/block-library/accordion-group' );
+
+	$p         = new WP_HTML_Tag_Processor( $content );
+	$autoclose = $attributes['autoclose'] ? 'true' : 'false';
```

Comment:
As we're not using `wp_json_encode`, we need to perform the boolean stringify ourselves.

Fixed in https://github.com/WordPress/gutenberg/pull/64119/commits/90a7c66db1174b44f49d35e41f314c866a200d5a.